src/PowerShell/Microsoft.WinGet.Client.Engine/Helpers/AppxModuleHelper.cs

```diff
@@ -94,6 +94,11 @@ internal class AppxModuleHelper
         private const string XamlPackage27 = "Microsoft.UI.Xaml.2.7";
         private const string XamlReleaseTag273 = "v2.7.3";
 
+        // WinGet Source
+        private const string WinGetSourceName = "Microsoft.Winget.Source";
+        private const string WinGetSourceMsixName = "source2.msix";
+        private const string WinGetSourceUrl = "https://cdn.winget.microsoft.com/cache/source2.msix";

```

Comment:
Is there a COM API for fetching source information? Since the winget source is a well-known source (can be disabled, but not removed), would it be a possible consideration to dynamically fetch the source URL through the API? That way, in the event a source3.msix ever becomes the default, this portion of the cmdlet would just work?

Reply:
Can we not have the file name twice?  Use string concatenation or formatting or whatever to only have `source2.msix` the one time.